packages/service-provider-core/src/writable.ts

```diff
@@ -319,7 +319,7 @@ export default interface Writable {
     collection: string,
     filter: Document,
     update: Document,
-    options?: UpdateOptions,
+    options?: UpdateOptions & { sort?: Document },
```

Comment:
matches i.e. https://github.com/mongodb/node-mongodb-native/commit/28857b79898d6a6040b7d48f3781ebc2710f0279#diff-1934939aca129a8507697ca1f4354d6fda9d4204d0e06577bd82e33eb6dbfa0bR267

@nbbeeken is there a reason why `{ sort }` ended up outside of `UpdateOptions` but is in `ReplaceOptions`?